src/ui/FileList/FileList.css

```diff
@@ -41,7 +41,7 @@
 }
 
 /* <element data-type="numeric"> */
```

Comment:
Nit: Can you update the element comment to correctly match the new selector? 

`/* <element className="filelistui-row" data-type="numeric"> */`